### ISSUE TITLE
[Discussion] Shaders: Blur shader

### ISSUE THREAD

**Issue by cbscribe** [kidscancode/godot_recipes#79]
Discussion for http://godotrecipes.com/shaders/blur/

**Comment by DriNeo** [comment]
Have you a project that use this shader please ? I have trouble to make it working.

**Comment by cbscribe** [comment]
No, but in the greyscale shader recipe there's an example of how you might use it:

http://godot_recipes.com/shaders/greyscale/
